api/v1/testkube.yaml

```diff
@@ -2100,6 +2100,34 @@ paths:
                 type: array
                 items:
                   $ref: "#/components/schemas/Problem"
+  /copy-files:
```

Comment:
is copy-files a valid resource name?

Reply:
I would use just `files` and `POST /files`

Reply:
We have agreed on using `--copy-files` in the CLI. `--file` is already used to give the test files as input. (and we also agreed that `--config-file` is too restrictive, as these files will not always be used as configuration files)

While I do agree that copy is a verb in itself, the closest adjective would be `copied`, but that implies it was already copied over. For consistency's sake I'd keep `/copy-files`, unless there is a better and simpler option.

Reply:
After a discussion in a private channel, we decided to go with /uploads for this endpoint. The code is being updated and tested rn